### ISSUE TITLE
self.cnn.add_module(base_model, globals().get(base_model)(pretrained=pretrained))

### ISSUE THREAD

**Issue by deep-practice** [BADBADBADBOY/pytorch.ctpn#13]

**Comment by BADBADBADBOY** [comment]
> globals()这个用法很妙，有详细一点的介绍吗

百度不香吗

**Comment by deep-practice** [comment]
明白了，是因为下面这几句，导入了这些文件里面的全局变量
from models.vgg import *
from models.resnet import *
from models.mobilenet import *
from models.shufflenetv2 import *

当时看见是灰色的就没有留意，还觉得很神奇。。。
